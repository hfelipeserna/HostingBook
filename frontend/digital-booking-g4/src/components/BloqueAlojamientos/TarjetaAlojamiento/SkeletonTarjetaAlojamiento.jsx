import styles from "./TarjetaAlojamiento.module.css";

export default function SkeletonTarjetaAlojamiento(){
    return (
      <div className={styles.tarjetaAlojamiento} data-aos="fade-up">
        <div className={`${styles.imagenAlojamiento} ${styles.skeleton}`}></div>
        <div className={styles.descripcionAlojamiento}>
          <div className={styles.informacionPrincipal}>
            <div className={styles.nombreAlojamiento}>
              <div className={styles.tipoYCalificacion}>
                <div
                  className={`${styles.infoLoader} ${styles.skeleton}`}
                ></div>
                <div
                  className={`${styles.infoLoader} ${styles.skeleton}`}
                ></div>
              </div>
              <div className={`${styles.infoLoader} ${styles.skeleton}`}></div>
            </div>
            <div className={styles.puntajeAlojamiento}>
              <div
                className={`${styles.puntajeNumerico} ${styles.infoLoader} ${styles.skeleton}`}
              ></div>
              <div
                className={`${styles.detalle} ${styles.infoLoader} ${styles.skeleton}`}
              ></div>
            </div>
          </div>
          <div className={`${styles.textLoader} ${styles.skeleton}`}></div>
          <div className={`${styles.textLoader} ${styles.skeleton}`}></div>
          <div className={`${styles.textLoader} ${styles.skeleton}`}></div>
          <div className={`${styles.inputLoader} ${styles.skeleton}`}></div>
        </div>
      </div>
    );
}