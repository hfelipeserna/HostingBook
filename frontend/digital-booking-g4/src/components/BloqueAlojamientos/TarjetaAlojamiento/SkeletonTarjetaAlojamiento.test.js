import SkeletonTarjetaAlojamiento from "./SkeletonTarjetaAlojamiento";
import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react"

describe("Skeleton Tarjeta Alojamiento test", function () {

    test("Renderiza Componente Por default", () => {
        const renderizacionSimple = render(<SkeletonTarjetaAlojamiento></SkeletonTarjetaAlojamiento>)
        expect(renderizacionSimple).toMatchSnapshot()
    })

})