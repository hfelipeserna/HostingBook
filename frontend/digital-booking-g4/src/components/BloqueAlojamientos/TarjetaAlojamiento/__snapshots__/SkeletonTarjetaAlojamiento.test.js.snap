// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton Tarjeta Alojamiento test Renderiza Componente Por default 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="tarjetaAlojamiento"
        data-aos="fade-up"
      >
        <div
          class="imagenAlojamiento skeleton"
        />
        <div
          class="descripcionAlojamiento"
        >
          <div
            class="informacionPrincipal"
          >
            <div
              class="nombreAlojamiento"
            >
              <div
                class="tipoYCalificacion"
              >
                <div
                  class="infoLoader skeleton"
                />
                <div
                  class="infoLoader skeleton"
                />
              </div>
              <div
                class="infoLoader skeleton"
              />
            </div>
            <div
              class="puntajeAlojamiento"
            >
              <div
                class="puntajeNumerico infoLoader skeleton"
              />
              <div
                class="detalle infoLoader skeleton"
              />
            </div>
          </div>
          <div
            class="textLoader skeleton"
          />
          <div
            class="textLoader skeleton"
          />
          <div
            class="textLoader skeleton"
          />
          <div
            class="inputLoader skeleton"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="tarjetaAlojamiento"
      data-aos="fade-up"
    >
      <div
        class="imagenAlojamiento skeleton"
      />
      <div
        class="descripcionAlojamiento"
      >
        <div
          class="informacionPrincipal"
        >
          <div
            class="nombreAlojamiento"
          >
            <div
              class="tipoYCalificacion"
            >
              <div
                class="infoLoader skeleton"
              />
              <div
                class="infoLoader skeleton"
              />
            </div>
            <div
              class="infoLoader skeleton"
            />
          </div>
          <div
            class="puntajeAlojamiento"
          >
            <div
              class="puntajeNumerico infoLoader skeleton"
            />
            <div
              class="detalle infoLoader skeleton"
            />
          </div>
        </div>
        <div
          class="textLoader skeleton"
        />
        <div
          class="textLoader skeleton"
        />
        <div
          class="textLoader skeleton"
        />
        <div
          class="inputLoader skeleton"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
